src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Services/AssemblyLoadContextWrapper.cs

```diff
@@ -33,7 +31,7 @@ private AssemblyLoadContextWrapper(AssemblyLoadContext assemblyLoadContext, Immu
             _logger = logger;
         }
 
-        public static bool TryLoadExtension(string assemblyFilePath, string? sharedDependenciesPath, ILogger? logger, [NotNullWhen(true)] out Assembly? assembly)
+        public static bool TryLoadExtension(string assemblyFilePath, ILogger? logger, [NotNullWhen(true)] out Assembly? assembly)
```

Comment:
@tmat would like you're review on this PR (but especially here).  Seems unnecessary to load shared deps from devkit now.  Smoke test on telemetry and hot reload works.

Reply:
Yeah, I think it'd even be incorrect to do so. We don't want to depend on versions of assemblies shipping in the DevKit.